src/services/updateZendeskTicket.ts

```diff
@@ -1,8 +1,41 @@
-export const updateZendeskTicket = async (
-  requestBody: string | null,
-  message: string
-) => {
-  console.log(
-    `Updating Zendesk Ticket. Request body was ${requestBody}. Message: ${message}`
-  )
+import https from 'node:https'
+import { makeHttpsRequest, base64Encode } from './httpsRequestUtils'
+import { retrieveZendeskApiSecrets } from './retrieveZendeskApiSecrets'
+
+export async function updateZendeskTicket(
+  eventBody: string | null,
+  validationMessage: string
+) {
+  if (!eventBody) {
+    console.log('No Zendesk info available. Cannot update ticket.')
+    return
+  }
+  const zendeskTicketInfo = JSON.parse(eventBody)
+  const secrets = await retrieveZendeskApiSecrets()
+  const options: https.RequestOptions = {
+    method: 'PUT',
+    hostname: process.env.ZENDESK_HOSTNAME,
+    path: `/api/v2/tickets/${zendeskTicketInfo.zendeskId}.json`,
```

Comment:
I think we should check for the absence of `zendeskId`, and if it's not there log that fact, and not make the API call. Remember we're dealing a potentially malformed request, so it could actually have this field missing.

Reply:
added a check for ZendeskId along with a test